### DISCUSSION TITLE
Can i change the width of NavigationDrawer?

### DISCUSSION THREAD

**Discussion by plusblanc** [flet-dev/flet#3654] [Q&A]
I want to make the NavigationDrawer wider because my app only works on Windows.
NavigationDrawer doesn't seem to have a “width” property.
Are there any way to resolve this issue?
A panel wider than the default width of this NavigationDrawer doesn't have to be a navigation drawer. It can be any panel (even if it's not a navigation drawer) that slides in horizontally from the left or right edge of a page to show primary destinations in an app.

**Comment by PRz-BigJ** [comment]
You've got an awser for your question?
'Cause I'm having a similar problem, but in my case what I want is to decrease width.
